apps/aechannel/test/aesc_fsm_SUITE.erl

```diff
@@ -170,7 +172,10 @@ groups() ->
       ]},
      {initiator_is_ga, [sequence], ga_sequence()},
      {responder_is_ga, [sequence], ga_sequence()},
-     {both_are_ga, [sequence], ga_sequence()}
+     {both_are_ga, [sequence], ga_sequence()},
+     {client_reconnect, [sequence],
+      [ client_reconnect_initiator
```

Comment:
This can be one test case.

Reply:
I am partial to reasonably specialized test cases, since debugging failing complex test cases can be quite time-consuming.